multiscale_phate/multiscale_phate.py

```diff
@@ -60,21 +75,39 @@ def __init__(
         self.gamma = gamma
         self.knn = knn
         self.n_jobs = n_jobs
+        self.NxTs = None
+        self.Xs = None
+        self.Ks = None
+        self.merges = None
+        self.Ps = None
+        self.diff_op = None
+        self.data_pca = None
+        self.pca_op = None
+        self.partition_clusters = None
+        self.dp_pca = None
+        self.epsilon = None
+        self.merge_threshold = None
+        self.gradient = None
+        self.levels = None
+
+
         super().__init__()
 
     def fit(self, X):
-        """Short summary.
+        """Builds a Diffusion Condensation tree to the data and computes
```

Comment:
First line of docstring should be short and fit on one line. Detailed description can follow after two linebreaks. e.g.

```
"""Compute Multiscale PHATE on data.

Builds a condensation tree on data and computes ideal
levels of granularity via gradient analysis.

Parameters
...
"""
```